rake/spec/default/70-st2_actions-integrity_spec.rb

```diff
@@ -4,7 +4,7 @@
     its(:exit_status) { is_expected.to eq 0 }
   end
 
-  describe command("st2 pack install github") do
+  describe command("st2 pack install xml") do
```

Comment:
https://github.com/stackstorm-exchange/stackstorm-github contains `actions`, `aliases`, `rules` and `sensors`, opposing to `xml` contains `actions` only.

Reply:
In this case I will change it to use ``github`` if ``mistral_enabled`` is True, otherwise use a different pack.

Reply:
What's wrong with github pack?

Reply:
It includes a Mistral workflow.

Reply:
It looks like this pack contains all the resources (minus sensors) and doesn't rely on a Mistral workflow - https://github.com/StackStorm-Exchange/stackstorm-hubot

Reply:
`hubot` would be good enough

Reply:
Pushed that change - I also think it's better than installing two different packs based if Mistral if available or not.